list_sorting.py

```diff
@@ -0,0 +1,39 @@
+#function to sort elements according to evens, odds and characters
+def list_sort(a):
+    if isinstance(a, list):
+        #list categories
+        word_cat = ['evens','odds','chars']
+        #sorted lis
+        sorted_dictionary = {}
+        char_list = []
+        evens_list = [] 
+        odds_list = []
+        for x in a:
+            if isinstance(x, str):
+                char_list.append(x)
+                
+            elif x % 2 == 1:
+                evens_list.append(x)
+                
+            elif x % 2 == 0:
+                odds_list.append(x)
+            else:
```

Comment:
I would advise you to remove this cuz it brings about a bug for example Try running your programm with the input below
```python
list_sort([4,9,7,8,True, 'd', 'g', False, '!'])
```

and see your output

Reply:
Have tried running the program with this list_sort([4,9,7,8,True, 'd', 'g', False, '!']) and it outputs this:
{'evens': [False, 4, 8], 'chars': ['!', 'd', 'g'], 'odds': [True, 7, 9]}